dbt/models/shared_columns.md

```diff
@@ -1325,6 +1325,12 @@ Indicates an outlier sale not used in modeling or reporting.
 
 This variable combines `sv_is_heuristic_outlier`
 with `sv_is_ptax_outlier` (using OR logic).
+
+A null value represents an observation that, due to
+filters on type of sale or time frame of sale, is
+excluded completely from the sales-val model pipeline
+and therefor does recieve a boolean value.
+
 NOTE: Outlier flags only exist for sales _after_ 2014.
```

Comment:
**[Suggestion, non-blocking]** Maybe we could make the connection here explicit:

```suggestion
NOTE: Outlier flags only exist for sales _after_ 2014.
Sales before 2014 will have a null value for this column.
```

Also, is the date range inclusive of 2014? If not, the above text should read "Sales _including and_ before 2014".